src/ontology/edits/oeo-shared.omn

```diff
@@ -4682,12 +4827,18 @@ https://github.com/OpenEnergyPlatform/ontology/pull/1041
 
 Add alternative label and improve definition:
 issue: https://github.com/OpenEnergyPlatform/ontology/issues/1455
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1562",
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1562
+
+change equivalence axiom
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1527
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1599",
         rdfs:label "electricity generation process"
     
     EquivalentTo: 
         OEO_00020003
-         and (OEO_00010235 some OEO_00000139)
+         and (OEO_00020259 some OEO_00000139)
+         and (OEO_00020257 only 
+            (OEO_00000007 or OEO_00000207 or OEO_00020040 or OEO_00020045 or OEO_00230020))
```

Comment:
If we do it by listing all non-electrical energies, then `nuclear binding energy` is missing here.